src/test/java/org/broadinstitute/hellbender/utils/mcmc/AdaptiveMetropolisSamplerUnitTest.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.hellbender.utils.mcmc;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Created by davidben on 11/24/15.
+ */
+public final class AdaptiveMetropolisSamplerUnitTest {
+    private static final double INITIAL_STEP_SIZE = 0.1;
+    private static final double INITIAL_GAUSSIAN_GUESS = 0;
+    private static final double INITIAL_BETA_GUESS = 0.5;
+    private static final int NUM_BURN_IN_STEPS = 1000;
+    private static final int NUM_SAMPLES = 5000;
+
+    @Test
+    public void testBeta() {
+        for (final double a : Arrays.asList(10, 20, 30)) {
+            for (final double b : Arrays.asList(10, 20, 30)) {
+
+                final double theoreticalMean = a / (a + b);
+                final double theoreticalVariance = a*b/((a+b)*(a+b)*(a+b+1));
+                final double standardDeviationOfMean = Math.sqrt(theoreticalVariance / NUM_SAMPLES);
+
+                final Function<Double, Double> logPDF = x -> (a - 1) * Math.log(x) + (b - 1) * Math.log(1 - x);
+                final AdaptiveMetropolisSampler sampler = new AdaptiveMetropolisSampler(INITIAL_BETA_GUESS, INITIAL_STEP_SIZE, 0, 1);
+                final List<Double> samples = sampler.sample(logPDF, NUM_SAMPLES, NUM_BURN_IN_STEPS);
+
+                final double sampleMean = samples.stream().mapToDouble(x -> x).average().getAsDouble();
+                final double sampleMeanSquare = samples.stream().mapToDouble(x -> x*x).average().getAsDouble();
+                final double sampleVariance = (sampleMeanSquare - sampleMean * sampleMean)*NUM_SAMPLES/(NUM_SAMPLES-1);
+
+                Assert.assertEquals(sampleMean, theoreticalMean, 10 * standardDeviationOfMean);
+                Assert.assertEquals(sampleVariance, theoreticalVariance, 10e-4);
+            }
+        }
+    }
+
+    @Test
+    public void testGaussian() {
```

Comment:
shoot, i think i figured out what was going on---there was a bug in my testing code!  i'd forgotten that i'd experimented with thinning in the sample method at one point (accepting only i % 10 == 0 samples), but then didn't change the number of samples going in the denominator of the standard error accordingly later (which accounts for the factor of ~1/3.  doh, sorry about that!

however, perhaps this suggests that we could overload the sample method to allow for thinning.  if so, then perhaps we should change the meaning of numSamples, numBurnIn, etc.  we can handle this in future PRs if we decide it's worth it.